jsonschema-validation.xml

```diff
@@ -550,6 +550,20 @@
                 </t>
             </section>
 
+            <section title="propertyNames">
+                <t>
+                    The value of "propertyNames" MUST be a valid JSON Schema.
+                </t>
+                <t>
+                    If the instance is an object, this keyword validates if every property name in the instance
+                    validates against the provided schema.
+                    Note the property name that the schema is testing will always be a string.
+                </t>
+                <t>
+                    A missing keyword is the same as an empty schema, or a schema testing that the type is "string".
```

Comment:
Same objection to this language as in PR #171, so this should be adjusted based on the resolution of that PR.